LeaderboardBackend/Models/Entities/User.cs

```diff
@@ -1,14 +1,27 @@
 using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
 using System.Text.Json.Serialization;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace LeaderboardBackend.Models.Entities;
 
+public enum UserRole : byte
```

Comment:
I don't think we need to explicitly specify `byte` here. In fact, I would avoid doing so as to not mess up the DB mapping.